### ISSUE TITLE
conversion error on version 2.02

### ISSUE THREAD

**Issue by Marcuzzo77** [potree/PotreeConverter#479]
Hi,

I downloaded the PotreeConverter_2.0.2_x64_windows.zip version. when with "cmd" in Windows 10 I try to run the command:

C:\Users\msw\ Documents\Potree\converter\PotreeConverter_2.0.2_x64_windows\PotreeConverter.exe C:\Users\cloud.ply -o C:\Users\potree

this is returned to me:

#threads: 4
#paths: 0

output attributes:
name offset size
===============================================
===============================================
                                               0
===============================================
cubicAABB: {
        "min": [inf, inf, inf],
        "max": [-nan (ind), -nan (ind), -nan (ind)],
        "size": [-inf, -inf, -inf]
}
#points: 0
total file size: 0.0 bytes
target directory: 'C: \ Users \ msw \ Documents \ Photogrammetry \ flasks \ potree'
maxPointsPerChunk: 0

=======================================
=== COUNTING
=======================================
tStartTaskAssembly: 0.000001s
countPointsInCells: 0.014786s
finished counting in 0s
=======================================
createLUT: 0.047673s

=======================================
=== CREATING CHUNKS
=======================================
distributePoints0: 0.000372s
distributePoints1: 0.000574s
finished creating chunks in 0s
=======================================

=======================================
=== INDEXING
=======================================

unfortunately,

 a subfolder and a file that I attach in the image appears in the destination folder.
There is no trace of the conversion.
Any idea?

PS with version 1.7 everything works fine

![Schermata 2020-11-23 alle 12 16 45](https://user-images.githubusercontent.com/26044319/99956325-22581e80-2d86-11eb-8574-8d5514ceadd6.png)

**Comment by midnight-dev** [comment]
Try converting your point cloud to an LAS format first. While PC 1.7 supports many input formats, version 2.0.0+ needs LAS or LAZ files.

**Comment by Marcuzzo77** [comment]
@midnight-dev ok importing in las format now works, this time however unlike version 1.7 I don't find the file in .html format.
I have three files:
- hierarchy.bin
- metadata.json
- octree.bin
how do i proceed for the conversion to the server?
tks

**Comment by midnight-dev** [comment]
Glad it works now. To generate an HTML & JavaScript package that's configured to use the conversion output, try adding this to the end of your command: `-p MyConversion`

I haven't yet tested the feature in the latest version, but let us know if you have any trouble. You should be able to customize the project name. I just used "MyConversion" because it's a simple placeholder.

**Comment by Marcuzzo77** [comment]
@midnight-dev Hi Ben, I added -p MyConversion to the end of the string but this time I only see one folder called "libs".
This is the result of the process:

#threads: 4
#paths: 1

output attributes:
name offset size
===============================================
position 0 12
intensity 12 2
return number 14 1
number of returns 15 1
classification 16 1
scan angle rank 17 1
user data 18 1
point source id 19 2
rgb 21 6
===============================================
                                              27
===============================================
cubicAABB: {
        "min": [-178.671000, -146.023000, 107.101000],
        "max": [189.450000, 222.098000, 475.222000],
        "size": [368.121000, 368.121000, 368.121000]
}
#points: 20'350'815
total file size: 504.6 MB

**Comment by midnight-dev** [comment]
Right, sorry about that. I'm going to check the feature a little bit later today.

In the meantime, check the core [Potree](https://github.com/potree/potree) project and grab one of the HTML files from the examples directory. Change the URLs to match your locations of all of the dependencies. Then just replace the string inside the call that loads the point cloud. Instead of loading a _cloud.js_ file, you have to pass in the location of _metadata.json_. The JavaScript will know how to load the new format, assuming you're using version 1.7 and above.
